zookeeper-3.9.yaml

```diff
@@ -91,28 +117,58 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/opt/bitnami/zookeeper/logs
           mkdir -p ${{targets.subpkgdir}}/opt/bitnami/scripts/zookeeper/
 
-          cp -r ${{targets.destdir}}/usr/share/java/zookeeper/bin/* ${{targets.subpkgdir}}/opt/bitnami/zookeeper/bin
-          ln -s /usr/share/java/zookeeper/lib ${{targets.subpkgdir}}/opt/bitnami/zookeeper/lib
+          cp -r ${{targets.destdir}}/apache-zookeeper-${{package.version}}-bin/bin/* ${{targets.subpkgdir}}/opt/bitnami/zookeeper/bin
+          ln -s /apache-zookeeper-${{package.version}}-bin/lib ${{targets.subpkgdir}}/opt/bitnami/zookeeper/lib
           cp -r conf/* ${{targets.subpkgdir}}/opt/bitnami/zookeeper/conf/
           cp -r conf/* ${{targets.subpkgdir}}/opt/bitnami/zookeeper/conf.default
 
           # create symlinks for both /entrypoint.sh and /run.sh to make it compatible with bitnami/zookeeper helm chart
           ln -sf /opt/bitnami/scripts/zookeeper/entrypoint.sh "${{targets.subpkgdir}}"/entrypoint.sh
           ln -sf /opt/bitnami/scripts/zookeeper/run.sh "${{targets.subpkgdir}}"/run.sh
     test:
+      environment:
+        contents:
+          packages:
+            - uuidgen
```

Comment:
is uuidgen required by the script? if yes; we should add it as runtime dep. If no, we can consider removing this.

Reply:
It's used in the tests to generate uuid while creating a node